docs/creature.md

```diff
@@ -9,58 +9,73 @@ Contains individual creature spawn data for each individual spawn of each indivi
+| [equipment_id][10]     | TINYINT      | UNSIGNED   |     | NO   | 1       |                |                                                                                     |
+| [position_x][11]      | FLOAT        | SIGNED     |     | NO   | 0       |                |                                                                                       |
+| [position_y][12]      | FLOAT        | SIGNED     |     | NO   | 0       |                |                                                                                       |
+| [position_z][13]      | FLOAT        | SIGNED     |     | NO   | 0       |                |                                                                                       |
+| [orientation][14]     | FLOAT        | SIGNED     |     | NO   | 0       |                |                                                                                       |
+| [spawntimesecs][15]   | INT          | UNSIGNED   |     | NO   | 120     |                |                                                                                       |
+| [wander_distance][16] | FLOAT        | SIGNED     |     | NO   | 5       |                | Dist in yards for random movement.                                                    |
+| [currentwaypoint][17] | MEDIUMINT    | UNSIGNED   |     | NO   | 0       |                | Storage used by core. "Always set as 0"                                               |
+| [curhealth][18]       | INT          | UNSIGNED   |     | NO   | 1       |                | Storage used by core. "Always set as 1"                                               |
+| [curmana][19]         | INT          | UNSIGNED   |     | NO   | 0       |                | Storage used by core. "Always set as 0"                                               |
+| [MovementType][20]    | TINYINT      | UNSIGNED   |     | NO   | 0       |                | 0 No movement, 1 random, 2 path                                                       |
+| [npcflag][21]         | INT          | UNSIGNED   |     | NO   | 0       |                |                                                                                       |
+| [unit_flags][22]      | INT          | UNSIGNED   |     | NO   | 0       |                |                                                                                       |
+| [dynamicflags][23]    | INT          | UNSIGNED   |     | NO   | 0       |                |                                                                                       |
 
 [1]: #guid
-[2]: #id
-[3]: #map
-[4]: #zoneId
-[5]: #areaId
-[6]: #spawnmask
-[7]: #phasemask
-[8]: #equipment_id
-[9]: #position_x
-[10]: #position_y
-[11]: #position_z
-[12]: #orientation
-[13]: #spawntimesecs
-[14]: #wander_distance
-[15]: #currentwaypoint
-[16]: #curhealth
-[17]: #curmana
-[18]: #movementtype
-[19]: #npcflag
-[20]: #unit_flags
-[21]: #dynamicflags
+[2]: #creature_id1
+[3]: #creature_id2
```

Comment:
Remove the underscore here. Otherwise links dont work.

It's fine for the headers but the links need to be without